packages/@okta/vuepress-site/docs/guides/custom-error-pages/main/index.md

```diff
@@ -26,15 +27,63 @@ Customize error pages as part of the sign-in flow.
+When using redirect authentication, you can create a unique sign-in experience. First, set up a [custom domain](/docs/guides/custom-url-domain/) and then create a [Widget style](/docs/guides/custom-widget/main/#style-the-okta-hosted-sign-in-widget) that matches your application's look and feel.
 
-However, if an error occurs during sign-in, Okta may need to display an error page to the user. To provide a seamless user experience, you can also customize the error page by using the code editor.
+If an error occurs during sign-in, Okta needs to display an error page to the user. You can use the code editor to customize the error page.
 
-> **Note:** A custom error page only appears when an app connects to Okta by using your custom domain. Otherwise, the default Okta error page appears when a critical error occurs or an application is misconfigured. See [Okta deployment models &mdash; redirect vs. embedded](/docs/concepts/redirect-vs-embedded/#redirect-authentication) for more information on Okta-hosted (redirect) functionality.
+> **Note:** A custom error page only appears when an app connects to Okta by using your custom domain. Otherwise, the default Okta error page appears when a critical error happens or you haven't properly configured an app. See [Okta deployment models&mdash;redirect vs. embedded](/docs/concepts/redirect-vs-embedded/#redirect-authentication) for more information on redirect functionality.
 
 ### Use the Brands API
 
-The [Brands API](/docs/reference/api/brands/) allows you to set icons, images, and colors across your Okta-hosted Sign-In Widget, error pages, email templates, and Okta End-User Dashboard all at once. You don't need to set a customized Okta URL domain. See [Customize your Okta experience with the Brands API](/docs/guides/customize-themes) for more information on this feature and how to use it.
+The [Brands API](/docs/reference/api/brands/) allows you to set icons, images, and colors across the following:
+
+- sign-in page (redirect authentication)
+- error pages
+- email templates
+- Okta End-User Dashboard
+
+You don't need to set a customized Okta URL domain to use the API.
+
+## Content Security Policy (CSP) for your custom domain
+
+Set up a [custom domain](/docs/guides/custom-url-domain/main/) and customize your [CSP](https://content-security-policy.com/) if you also want to customize the sign-in page or error pages. CSP customizations only take effect on custom domains.
+
+To analyze and detect potentially malicious IP addresses that seek to bypass your CSP, use [Okta ThreatInsight](https://help.okta.com/okta_help.htm?type=oie&id=ext-about-threatinsight).
+
+### Known limitations
+
+* Don't use `meta` tags to customize the CSP. `meta` tags impact the overall policy. It's easier to control CSP customizations by adding trusted origin URLs in the Admin Console.
```

Comment:
@bhagyaprabhakar-okta How's this?

Reply:
I worry about the imperative tone of "Don't use meta tags".  All these days, when we did not have this feature we did say use 'meta' tags if you need to.  CC @bryanhonan-okta and @mihaiiacob-okta  for comments.